src/libguac/guacamole/user.h

```diff
@@ -493,6 +493,30 @@ guac_user* guac_user_alloc();
 void guac_user_free(guac_user* user);
 
 /**
+ * Handles the portion of a user's Guacamole protocol handshake following the
+ * "select" instruction all subsequent I/O, automatically populating the
```

Comment:
Nooooo ... my beautiful beautiful sentence ...

I will ... uh ... try to make this make sense.